File: TeamF_Api/DAL/Entity/Comment.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TeamF_Api.DAL.Entity
{
    public class Comment
    {
        public int Id { get; set; }
        public virtual User User { get; set; }
        public Guid UserId { get; set; }
        public string CommentText { get; set; }
        public virtual CaffEntity CaffEntity { get; set; }
        public int CaffEntityId { get; set; }
        public DateTime DateTime { get; set; }
    }
}

File: compsecfrontend/compSecFrontEnd/src/app/api/models/comment.ts
/* tslint:disable */
/* eslint-disable */
import { CaffEntity } from './caff-entity';
import { User } from './user';
export interface Comment {
  caffEntity?: CaffEntity;
  caffEntityId?: number;
  commentText?: null | string;
  dateTime?: string;
  id?: number;
  user?: User;
  userId?: string;
}
